__tests__/cron/updateCurrentStreak.ts

```diff
@@ -81,30 +80,156 @@ describe('updateCurrentStreak cron', () => {
       .getRepository(UserStreak)
       .update({ userId: '1' }, { lastViewAt: new Date('2024-06-21') });
     // This is a Monday
-    jest
-      .useFakeTimers({
-        doNotFake: [
-          'hrtime',
-          'nextTick',
-          'performance',
-          'queueMicrotask',
-          'requestAnimationFrame',
-          'cancelAnimationFrame',
-          'requestIdleCallback',
-          'cancelIdleCallback',
-          'setImmediate',
-          'clearImmediate',
-          'setInterval',
-          'clearInterval',
-          'setTimeout',
-          'clearTimeout',
-        ],
-      })
-      .setSystemTime(new Date('2024-06-24'));
+    jest.useFakeTimers({ doNotFake }).setSystemTime(new Date('2024-06-24'));
     await expectSuccessfulCron(cron);
     const streak = await con
       .getRepository(UserStreak)
       .findOneBy({ userId: '1' });
     expect(streak.currentStreak).toBe(1);
   });
+
+  describe('incorporating streaks recovery', () => {
+    beforeEach(() => {
+      nock('http://localhost:5000').post('/e').reply(204);
```

Comment:
Why do you need this one here?

Reply:
The worker fails without this. The existing cron might have been calling this at the end.